canvasapi/course.py

```diff
@@ -438,6 +439,41 @@ def create_late_policy(self, **kwargs):
 
         return LatePolicy(self._requester, late_policy_json["late_policy"])
 
+    def create_lti_resource_link(self, url, title=None, custom=None, **kwargs):
+        """
+        Create a new LTI resource link.
+
+        :calls: `POST /api/v1/courses/:course_id/lti_resource_links \
+        <https://canvas.instructure.com/doc/api/lti_resource_links.html#method.lti/resource_links.create>`_
+
+        :param course_id: The ID of the course.
+        :type course_id: `int`
+        :param url: The launch URL for the resource link.
+        :type url: `str`
+        :param title: The title of the resource link.
+        :type title: `str`, optional
+        :param custom: Custom parameters to send to the tool.
+        :type custom: `dict`, optional
+
+        :rtype: :class:`canvasapi.lti_resource_link.LTIResourceLink`
+        """
+
+        if not url:
+            raise RequiredFieldMissing("The 'url' paramter is required.")
```

Comment:
Misspelling, but also for consistency with the other `RequiredFieldMissing` exceptions, I'd change to this.

```suggestion
            raise RequiredFieldMissing("url is required as a str.")
```